iocore/eventsystem/UnixEThread.cc

```diff
@@ -161,6 +162,7 @@ EThread::process_event(Event *e, int calling_code)
         EventQueueExternal.enqueue_local(e);
       }
     } else if (!e->in_the_prot_queue && !e->in_the_priority_queue) {
+      MUTEX_RELEASE(lock);
```

Comment:
It looks like the `lock` is already released in line 154 for this case.

Reply:
Yes, I'll fix this.